src/pages/components/Product.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './product.scss';
+const Product = ({ prdlist }) => {
```

Comment:
```suggestion
import './product.scss';

const Product = ({ prdlist: { image, name, discription, color, price } }) => {
```
가독성을 위해 한칸 띄워주세요!

추가로 구조분해 할당을 이렇게 한번 더 진행할수도 있습니다!